md/sv2-workshop.md

```diff
@@ -216,7 +221,11 @@ assuming `$TP` is the path to `bitcoind`
 
 ```
 cd bitcoin-sv2
-$TP -datadir=$HOME/.bitcoin-sv2-workshop -signet -sv2 -sv2port=8442
+$TP -datadir=$HOME/.bitcoin-sv2-workshop -signet -sv2
+```
+⚠️ If you grabbed a binary from releases and you are on a **macOS x86-64 (Intel)**, you would probably have it in your downloads dir, so do this:
```

Comment:
they will also have to do codesign, which consists of two steps:

```
codesign -s - Bitcoin-Qt.app
```

then right click `Bitcoin-Qt.app` and click `Open`

but as stated on this https://github.com/plebhash/sv2-workshop/pull/3#discussion_r1633786444, I'm still skeptical that `Bitcoin-Qt` is going to be a viable path

@Sjors could you send me the compiled `bitcoind` and `bitcoin-cli` you have on your x86-64 mac?

that way I can manually upload them to https://github.com/plebhash/bitcoin/releases/ and completely remove the need for Qt, which seems like a very suboptimal path